build.gradle

```diff
@@ -0,0 +1,53 @@
+apply plugin: 'java'
+apply plugin: 'application'
+apply plugin: 'pmd'
+
+group 'com.github'
+version '1.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation group: 'net.sourceforge.pmd', name: 'pmd-java', version: '6.29.0'
```

Comment:
it is fine to skip explicit "group"/"name"

Reply:
Do you mean to change it to
`implementation "net.sourceforge.pmd:pmd-java:6.29.0"`

Reply:
yep